mainnet/pirin-1/protocols/osmosis-osmosis-atom.json

```diff
@@ -0,0 +1,45 @@
+{
+    "dex": "osmosis",
+    "dex_network": "OSMOSIS",
+    "lpn_ticker": "ATOM",
+    "stable_currency_ticker": "USDC_NOBLE",
+    "lease_currencies_tickers": [
+        "USDC_NOBLE"
+    ],
+    "payment_only_currencies_tickers": [
+        "ALL_SOL",
+        "ST_ATOM",
+        "OSMO",
+        "ST_OSMO",
+        "WETH",
+        "AKT",
+        "INJ",
+        "AXL",
+        "ALL_BTC",
+        "ALL_ETH"
```

Comment:
[Style] I would suggest keeping the currencies sorted in alphabetical order for consistency and easier modifications later.